### ISSUE TITLE
Adding a manga to library from Global Search / Browse doesn't instantly update it to show "In Library"

### ISSUE THREAD

**Issue by curche** [tachiyomiorg/tachiyomi#7418]
### Steps to reproduce

1. Global Search (or Search in a source works too) for anything from any source. Eg: Search Ayashimon (from MangaDex)
2. Open the manga and add it to library
3. Press Back to reach Global Search. Notice that there is no "In Library" blurring shown for the manga you just added to the library
4. Go back to library to see if manga was actually added (spoiler: it is)
5. Now go Global Search for the manga again

### Expected behavior

Manga added to library should show up with a shadowed cover and "in library" badge
This should show up instantly

### Actual behavior

During Step 3, Manga doesn't have "in library" button
After Step 5, it does show up with the "in library" badge

### Tachiyomi version

r4524

### Android version

Android 12

### Device

jasmine_sprout

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.13.4](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by zbue** [comment]
It also happens when favoriting manga from source by long click. Seems like https://github.com/tachiyomiorg/tachiyomi/commit/cb1830d74773544c6063371df74b268dab0c85fa is the cause?

**Comment by KasukeLp23** [comment]
One question. Will this bug fix it? Because it's still the same. Sorry for the insistence.
